Add pet tab fixes

dailenperezFIU opened the pull request:
Hey, guys check out my drop-down changes and see if you're able to add a pet successfully. Please do not delete this branch yet as I want to add things such as a flag to avoid asking male dogs if they are pregnant.

dailenperezFIU committed:
changed the drop down style but now trouble shooting an error.

dailenperezFIU committed:
Added functionality to new drop downs

Danny393 commented:
This looks good, only issues is that the react-native-material-dropdown-v2 package is outdated and no longer being updated, and it would be best to look to other dropdown packages that are still being supported/updated. Be mindful to reset the yarn.lock and package.json files to their previous versions before trying out other dropdwon packages, to prevent dead packages in the project.

I would advise trying out other dropdown packages so that we do not have to deal with warnings long term.

dailenperezFIU committed:
Removed comments and format code.

Danny393 commented on src/home/pets/AddPets.js:
```diff
@@ -90,328 +142,352 @@ export default class AddPets extends React.Component<> {
+        // const { shift } = this.state;
+
+        return ( <
+            ScrollView style = {
```
This styling of the components makes it difficult to follow the code, reverting back to the old formatting would be best

Danny393 commented on src/home/pets/AddPets.js:
```diff
@@ -90,328 +142,352 @@ export default class AddPets extends React.Component<> {
+                false
+            } >
+            <
+            LinearGradient colors = {
```
The LinearGradient component here can be removed to prevent a merge conflict or make the merge conflict easier to fix

dailenperezFIU committed:
Updated formatting and removed Linear Gradient.

dailenperezFIU committed:
Merge branch 'main' into addPetTabFixes

Danny393 commented on src/home/pets/AddPets.js:
```diff
@@ -1,428 +1,340 @@
+                    }
+                
+                    ]}
+                    value ={pregnancyPossible} 
```
this line is causing a crash

dailenperezFIU committed:
Update AddPets.js

Removed line that was causing crash and re-ran the app to make sure it does not crash anymore

Danny393 committed:
using new dropdown

Danny393 committed:
resolving merge conficts
